### PR TITLE
[doc] Document pre-requsite for PTR records

### PR DESCRIPTION
**Pull request by mansikulkarni96**
This PR adds a bullet point stating address used in the configMap must have a PTR record resolving to the instance host name.

### CODE REVIEW

**Review by codyhoag — APPROVED** [review]
LGTM. Will add to official docs

**Review by jrvaldes — APPROVED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ Any Windows instances that are to be attached to the cluster as a node must fulf
 A ConfigMap named `windows-instances` must be created in the WMCO namespace, describing the instances that should be
 joined to a cluster. The required information to configure an instance is:
 * An address to SSH into the instance with. This can be a DNS name or an ipv4 address.
+    * A PTR record must exist corresponding to the address used in the configMap resolving to the instance host name for
```
Comment (sebsoto) [inline README.md:51]: configMap -> ConfigMap
host name -> hostname

Comment (sebsoto) [reply README.md:51]: This doesnt seem like it should be a sub-bullet. This is separate from having address to SSH into.
Shouldnt this go into the BYOH instance pre-req doc?

**Review by sebsoto — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aravindhp) [reply README.md:51]: Good point, @sebsoto. +1

**Comment by aravindhp** [comment]
@mansikulkarni96 are you planning on reflecting https://github.com/openshift/windows-machine-config-operator/pull/584 in this PR? It seems that PR was also targeting BYOH pre-reqs so they can be combined.

**Review by aravindhp — APPROVED** [review]
/approve

**Pull request merged**